examples/test_cpp/main.cc

```diff
@@ -19,12 +19,12 @@ po::variables_map process_cmd_line(const int argc, char** argv) {
 	  ("features,x", po::value<size_t>()->default_value(10), "Features count")
     ("actions,a", po::value<size_t>()->default_value(2), "Number of actions")
     ("experiment_name,e", po::value<std::string>()->required(), "experiment name")
-    ("perf,p", "if it is perf test (otherwise - validity)")
```

Comment:
Is this tool changing?

Reply:
noone is using validity mode - and it was broken since flatbuff introduction. Better to remove  now and re-do if needed.

Reply:
Sounds good, I agree